### PR TITLE
Feature/alamofire extension changes

### PR DESCRIPTION
**Pull request by chriscombs**
Removed default unwrapper per #124

Note this is a breaking change

### CODE REVIEW

**Comment by mariusc** [comment]
If it's a breaking change, we should update the README as well. 

We can probably also consider making a migration guide. Otherwise, just write about the migration steps in the notes for the new release what's new.

File: Serpent/Serpent/Classes/Extensions/AlamofireExtension.swift [hunk 1]
```diff
@@ -68,13 +62,11 @@ public extension Alamofire.DataRequest
...
-     The default implementation checks for a "data" field in the JSON response, then checks for a field
-     with the same name as the target model. If not found, it passes the JSON response straight through.
-     
+
      - returns: The request
      */
     @discardableResult
-    public func responseSerializable<T:Decodable>(_ completionHandler: @escaping (DataResponse<T>) -> Void, unwrapper:@escaping Parser.Unwrapper = Parser.defaultUnwrapper) -> Self {
+    public func responseSerializable<T:Decodable>(_ completionHandler: @escaping (DataResponse<T>) -> Void, unwrapper:@escaping Parser.Unwrapper) -> Self {
         let serializer = Parser.serializer( {
```
Comment (dominik-hadl) [inline Serpent/Serpent/Classes/Extensions/AlamofireExtension.swift]: Space after colon is missing here :)

File: Serpent/Serpent/Classes/Extensions/AlamofireExtension.swift [hunk 2]
```diff
@@ -97,13 +89,11 @@ public extension Alamofire.DataRequest
...
      
      - parameter unwrapper: A closure that extracts the data to be parsed from the JSON response data.
-     The default implementation checks for a "data" field in the JSON response, then checks for a field
-     with the same name as the target model. If not found, it passes the JSON response straight through.
      
      - returns: The request
      */
 	@discardableResult
-    public func responseSerializable<T:Decodable>(_ completionHandler: @escaping (DataResponse<[T]>) -> Void, unwrapper:@escaping Parser.Unwrapper = Parser.defaultUnwrapper) -> Self {
+    public func responseSerializable<T:Decodable>(_ completionHandler: @escaping (DataResponse<[T]>) -> Void, unwrapper:@escaping Parser.Unwrapper) -> Self {
```
Comment (dominik-hadl) [inline Serpent/Serpent/Classes/Extensions/AlamofireExtension.swift]: Space after colon is missing here :)

File: Serpent/SerpentTests/Tests/AlamofireExtensionTests.swift [hunk 3]
```diff
@@ -41,7 +43,7 @@ class AlamofireExtensionTests: XCTestCase {
 			}
 		}
 		if let url = urlForResource(resource: "NetworkModel") {
-			manager.request(url, method: .get).responseSerializable(handler)
+            manager.request(url, method: .get).responseSerializable(handler, unwrapper: defaultUnwrapper)
 		}
```
Comment (dominik-hadl) [inline Serpent/SerpentTests/Tests/AlamofireExtensionTests.swift:47]: Spacing got mangled somehow.

**Comment by chriscombs** [comment]
Updated with fixes from @nickskull

**Review by dominik-hadl — APPROVED** [review]
Looks good! We should squash and merge.

**Pull request merged**